DesignSafe-CI/portal
# Task/WC-119: Use a Django group to manage project admins

## Overview: ##
Updates the process for managing projects as an admin user.
Before: An admin user would impersonate a project member and manage projects as that user.
After: The admin user is added to a "Project Admin" group in Django. Members of this group can view/edit/publish all projects and can interact with project file systems using the service account.

## PR Status: ##


## Related Jira tickets: ##

* [WC-119](https://tacc-main.atlassian.net/browse/WC-119)

## Summary of Changes: ##

## Testing Steps: ##
1. Create a group called `Project Admin` (case-sensitive) in the Django admin interface (https://designsafe.dev/admin/auth/group/)
2. Add your user to this group
3. Go to My Projects. You should see a list of all projects in the database, not just those tied to your user account.
4. Confirm that you have READ access to files in arbitrary projects.

## UI Photos:

## Notes: ##
